### PR TITLE
Adds AI using the campaign tracker module

### PR DESCRIPTION
**Pull request by Spoffy**
Adds AI to the mission using a hacked-in copy of the tracker module from the campaign.

Intended only as a temporary solution, until we get the Tour AI spawning implemented.

### CODE REVIEW

File: game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    Returns:
+        Nothing
+
+    Example(s):
+        [_currentMission] call vgm_s_fnc_missions_despawnMission;
+ */
+
+params ["_mission"];
+
+terminate (_mission get "overlord_fsm");
```
Comment (veteran29) [inline game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf]: `terminate` does not work on FSMs, FSMs do not have a "handle" like  scripts. It's an ID (number).

Currently overlord does not have a way to stop it but I'll add FSM variable to do that.
```suggestion
(_mission get "overlord_fsm") setFSMVariable ["_forceStop", true];
```

**Review by veteran29 — APPROVED** [review]

File: game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf (hunk 1 above)
Comment (veteran29) [reply game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf]: Added required feature in `rMASTER16067`

Comment (Spoffy) [reply game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf]: Perfect, I'll merge this for now and then open another PR with the stop feature.

Thanks!

Comment (veteran29) [reply game/functions/systems/missions/server/internal/fn_missions_despawnMission.sqf]: You could add it in this PR, support for stopping the FSM will be in internal build tomorrow so...

**Pull request merged**